packages/react/src/ErrorBoundary.tsx

```diff
@@ -33,34 +33,57 @@ export interface ErrorBoundaryFallbackProps<TError extends Error = Error> {
   reset: () => void
 }
 
-export interface ErrorBoundaryProps extends PropsWithDevMode<PropsWithChildren, ErrorBoundaryDevModeOptions> {
-  /**
-   * an array of elements for the ErrorBoundary to check each render. If any of those elements change between renders, then the ErrorBoundary will reset the state which will re-render the children
-   */
-  resetKeys?: unknown[]
-  /**
-   * when ErrorBoundary is reset by resetKeys or fallback's props.reset, onReset will be triggered
-   */
-  onReset?(): void
-  /**
-   * when ErrorBoundary catch error, onError will be triggered
-   */
-  onError?(error: Error, info: ErrorInfo): void
-  /**
-   * when ErrorBoundary catch error, fallback will be render instead of children
-   */
-  fallback: ReactNode | FunctionComponent<ErrorBoundaryFallbackProps>
-}
+type Condition = (error: Error) => boolean
```

Comment:
```suggestion
type ErrorBoundaryErrorValidator = (error: Error) => boolean
```

@minsoo-web @okinawaa I'm thinking for naming it as ErrorBoundaryErrorValidator? This interface will be used like below. If you have idea, suggest it please

```tsx
const Exmaple = () => {
  return (
    <ErrorBoundary catchOn={[isAxiosError]}>
      Children
    </ErrorBoundary>
  )
}
```